api/src/main/kotlin/org/jetbrains/kotlinx/dl/api/core/callback/TimeStopping.kt

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2021 JetBrains s.r.o. and Kotlin Deep Learning project contributors. All Rights Reserved.
+ * Use of this source code is governed by the Apache 2.0 license that can be found in the LICENSE.txt file.
+ */
+
+package org.jetbrains.kotlinx.dl.api.core.callback
+
+import org.jetbrains.kotlinx.dl.api.core.history.EpochTrainingEvent
+import org.jetbrains.kotlinx.dl.api.core.history.TrainingHistory
+
+/**
+ * Callback for stopping training of a model when a specified amount of time has passed.
+ *
+ * This callback could be used to limit the training time of a model to a specified maximum duration.
+ * When the specified amount of time passes, the training of the model is stopped (if it has not already been
+ * finished or stopped) at the end of current epoch.
+ *
+ * @property [seconds] Maximum amount of time before stopping training.
+ * @property [verbose] Verbosity mode.
+ */
+public class TimeStopping(
+    private val seconds: Int = 86400,
```

Comment:
Add TODO here: upgrade to https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.time/-duration/ after upgrade to Kotlin 1.5 (1.4.30 at this moment)

Reply:
I added the TODO note; however, I am not sure if expecting and working with `Duration` has more benefit than simply using seconds value (from user perspective).